LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-center-safe.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<title>Tests whether safe alignment works correctly on the anchor element.</title>
+<link rel="match" href="anchor-center-safe-ref.html">
+<link rel="help" href="https://drafts.csswg.org/css-anchor-position-1/#anchor-center">
+<div class="anchor">Anchor</div>
+<div class="infobox">
+  <p>Anchored element</p>
+</div>
+<style>
```

Comment:
Probably the `<style>` element should go before the content, since it belongs in the `<head>` which comes before `<body>` content.